src/agent.cpp

```diff
@@ -364,28 +364,52 @@ FeatureExtractor* Agent::getFeatureExtractor(feature_set_t feature_set_indx,
   }
 }
 
-status_t Agent::getGameStatus(const rcsc::AudioSensor& audio_sensor,
+std::vector<int> Agent::getGameStatus(const rcsc::AudioSensor& audio_sensor,
                                   long& lastTrainerMessageTime) {
+  std::vector<int> status;
   status_t game_status = IN_GAME;
+  int playerIndex = -1;   // Keeps track of which defender stopped the shot
+  int playerTeam = 1;     // 1 = offense, -1 = defense, 0 = no possession
```

Comment:
All the lines that set playerTeam should use the SideID enum rather than the integer.